config.json

```diff
@@ -1,9 +1,20 @@
 {
   "language": "VB.NET",
   "active": false,
-  "blurb": "",
+  "blurb": "VB.NET is an object-oriented programming language implemented on the .NET Framework.",
   "checklist_issue": 3,
   "exercises": [
+    {
+      "slug": "two-fer",
+      "uuid": "57f02d0e-7b75-473b-892d-26a7d980c4ce",
```

Comment:
This uses the same UUID as the C# one. Each exercise's UUID must be globally unique though. You can generate a new v4 UUID at https://www.uuidgenerator.net/version4